spec/Body/Body.vspec

```diff
@@ -90,11 +90,102 @@ Windshield.Wiping:
   type: branch
   description: Windshield wiper signals.
 
-Windshield.Wiping.Status:
+######################## Idea ############################################
+# Introduce two sub-branches, one intended for functionality typically directly controllable by driver/user of vehicle,
+# Another with more low level functionality, typically not directly accessible by driver/user of vehicle, but rather
+# intended to be controlled by a vehicle function considering not only driver/user request but also other information
+# like speed, rain intensity and so on
+#
+####################################################################
+
+Windshield.Wiping.UserControl:
+  type: branch
+  description: High Level signals typically available to user/driver to control wiper functionality
+  
+Windshield.Wiping.UserControl.RequestedMode:
   datatype: string
   type: actuator
   allowed: ["off", "slow", "medium", "fast", "interval", "rainsensor"]
-  description: Wiper status.
+  description: Wiper mode requested by user/driver.
+  
+Windshield.Wiping.UserControl.RequestedIntensity:
+  datatype: uint8
+  type: actuator
+  description: Relative intensity/sensitivity for interval and rain sensor mode as requested by user/driver.
+               0 - wipers inactive.
+               1 - minimum intensity (lowest frequency/sensitivity, longest intervals).
+               2/3/4/... - higher intensity (higher frequency/sensitivity, shorter interval).
+               Maximum value supported is vehicle specific.
+
+
+Windshield.Wiping.SystemControl:
+  type: branch
+  description: Low Level signals to control behavior of wiper system in detail.
+               These signals are typically not directly available to the user/driver of the vehicle.
+
+
+Windshield.Wiping.SystemControl.Mode:
+  datatype: string
+  type: actuator
+  allowed: [
+    STOP, # Wipers finish current wiping cycle and then moves to parking position
+    STOP_APP, # Wipers finish current wiping cycle and then moves to alternative parking position
```

Comment:
We probably need to explain that APP is done to lessen wear of the wipers, otherwise it is unclear to 99% of the population